torch/quantization/fake_quantize.py

```diff
@@ -170,7 +170,23 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
             key = prefix + name
             if key in state_dict:
                 val = state_dict[key]
-                setattr(self, name, val)
+                # Custom handling to allow loading min_vals or max_vals
+                # of size N into uninitialized buffers of size 0. The
+                # buffers are resized here, and the values are copied in
+                # the default state_dict loading code of the parent.
+                if name == 'scale':
+                    self.scale.resize_(val.shape)
+                else:
+                    assert name == 'zero_point'
+                    self.zero_point.resize_(val.shape)
+                # For torchscript module we need to update the attributes here since we do not
+                # call the `_load_from_state_dict` function defined module.py
+                if torch.jit.is_scripting():
```

Comment:
looks like https://github.com/pytorch/pytorch/pull/44537/files does not have this code, should we add it there as well

Reply:
yes, it was added later to that code path as well: https://github.com/pytorch/pytorch/blob/master/torch/quantization/observer.py#L590